CMakeLists.txt

```diff
@@ -679,6 +679,10 @@ if(SDL_ASSEMBLY)
   set(HAVE_ASSEMBLY TRUE)
 
   if(SDL_MMX)
+   if(APPLE_MULTIARCH AND SDL_CPU_X64)
+    set(HAVE_MMX TRUE)
+    message(STATUS "SIMD: MMX enabled for Apple multi-arch config")
+   else()
```

Comment:
I updated #12731, which keeps the tests.

Reply:
Testing is pretty much meaningless in apple multi-arch configs.

Reply:
... and NEON check seems to always fail as it it now in #12731